web-server/v0.4/src/routes/Dashboard/Summary.js

```diff
@@ -18,6 +18,34 @@ const tabList = [
     key: 'tools',
     tab: 'Tools & Parameters',
   },
+  {
+    key: 'toc',
+    tab: 'Table of Content',
+  },
+];
+
+const DirectoryTree = Tree.DirectoryTree;
```

Comment:
`DirectoryTree` is not being used within this route so we should remove this reference.

Reply:
@gurbirkalsi will resolve this.